samples/matter/common/src/bridge/ble_bridged_device.h

```diff
@@ -46,8 +44,11 @@ class BLEBridgedDeviceProvider : public BridgedDeviceDataProvider {
 
 	BLEBridgedDevice &GetBLEBridgedDevice() { return mDevice; }
 	void SetConnectionObject(bt_conn *conn) { mDevice.mConn = conn; }
+	void RemoveConnectionObject() { mDevice.mConn = nullptr; }
+	bool IsInitialized() { return mDevice.mIsInitialized; }
+	void InitializationDone() { mDevice.mIsInitialized = true; }
 
-	bt_addr_le_t GetBtAddress() { return mDevice.mAddr; }
+	bt_addr_le_t *GetBtAddress() { return &mDevice.mAddr; }
```

Comment:
Why do we want to return member address instead of value? It potentially allows to access and modify the `mDevice` member by external parties. I think it would be safer to allow only provider to modify its `mDevice`.